activities/tests/test_detail.py

```diff
@@ -17,10 +18,10 @@ def test_future_activity(self):
         """Future/Upcoming activity should be accessible."""
         activity = create_activity("past", 1)
         response = self.client.get(urls.reverse("activities:detail", args=[activity.id]))
-        self.assertEqual(response.status_code, 200)
+        self.assertJSONEqual(response.content, activity_to_json(activity, True))
 
     def test_with_max(self):
         """Maximum number of participant should be shown in detail page if it has been set."""
         activity = create_activity("test1", 1, 1, 10)
         response = self.client.get(urls.reverse("activities:detail", args=[activity.id]))
-        self.assertContains(response, f"/ {activity.max_people}")
+        self.assertJSONEqual(response.content, activity_to_json(activity, True))
```

Comment:
How can this ensure that max_people will show? Or it just js thing I'm not sure about that?

Reply:
It will compare everything in the JSON and respose that we get. JSON contains id, name, detail, people, max_people, date and can_join. Response is what we get from detail view based on JSON format.